js/__internal/m_draggable.ts

```diff
@@ -769,6 +806,12 @@ const Draggable = (DOMComponent as any).inherit({
 
     return $scrollable;
   },
+  _getScrollableScrollTop() {
+    return this._getScrollable($(this.element()))?.scrollTop() ?? 0;
+  },
+  _getScrollableScrollLeft() {
```

Comment:
Let's add the empty line here:
```suggestion
  },
  
  _getScrollableScrollLeft() {
```